src/components/form/Select.tsx

```diff
@@ -21,8 +22,8 @@ const StyledSelect = styled.select`
   }
 `;
 
-function Select({ ...props }: SelectProps) {
```

Comment:
Would it bring any value to keep using `function` for our components?

```ts
function Input(props, ref) {
  return <StyledInput ref={ref} {...props} />;
}

const ForwardedInput = React.forwardRef(Input);

export default ForwardedInput;
```

Reply:
Not sure

Reply:
It seemed to cause console errors so i decided on not including this in this pr